.circleci/common.py

```diff
@@ -39,14 +39,14 @@ def get_airflow_version(ac_version):
     ("1.10.15-4", ["buster"]),
     ("2.0.0-10", ["buster"]),
     ("2.0.2-6", ["buster"]),
-    ("2.1.0-7.dev", ["buster"]),
-    ("2.1.1-6.dev", ["buster"]),
-    ("2.1.3-4.dev", ["buster"]),
-    ("2.1.4-4.dev", ["buster"]),
-    ("2.2.0-5.dev", ["bullseye", "buster"]),
-    ("2.2.1-3.dev", ["bullseye", "buster"]),
-    ("2.2.2-2.dev", ["bullseye", "buster"]),
-    ("2.2.3-2.dev", ["bullseye"]),
+    ("2.1.0-7", ["buster"]),
+    ("2.1.1-6", ["buster"]),
+    ("2.1.3-4", ["buster"]),
```

Comment:
Worth adding these versions in the PR title too or mentioning the range 2.1.0 to 2.2.2

Reply:
You're too quick :)